packages/extension-ui/src/components/Address.tsx

```diff
@@ -62,13 +65,12 @@ function Address ({ address, children, className, genesisHash, name }: Props): R
 
     const [formatted, account, chain] = recodeAddress(address, accounts, genesisHash);
 
-    setFormatted(formatted);
+    setFormatted(formatted.slice(0, 35) + '...');
```

Comment:
test?

Reply:
Looks like it works already in the original extension. Maybe it was handled in CSS?